packages/shared/internalAct.js

```diff
@@ -4,36 +4,27 @@
  * This source code is licensed under the MIT license found in the
```

Comment:
`/shared/` is ideally for nothing. Since these are only shared amongst Fiber renderers (e.g. no Fizz, DevTools, isomorphic), can it go into `react-reconciler/src/` instead? That's typically where

Reply:
I guess the intention is to move this to isomorphic next. That makes me think that maybe `act` on isomorphic must consider Fizz too.

Reply:
I ended up moving it to `jest-react` since that's our grab bag of testing helpers (all the `toFlush` helpers live there; nobody but us uses it).

I just needed to put it somewhere that's accessible in the build tests. Also could have put it in the `scripts` directory.

Reply:
Btw this is only the "internal" version of act.  The one that works by mocking the Scheduler. Not the public one that uses dev-time logic.